control-plane/src/modules/workflows/workflow-messages.ts

```diff
@@ -332,79 +343,40 @@ export const toAnthropicMessage = (message: TypedMessage): Anthropic.MessagePara
   }
 };
 
-const validateMessage = (message: Pick<RunMessage, "data" | "type">): TypedMessage => {
-  switch (message.type) {
-    case "agent": {
-      assertAgentMessage(message);
-      break;
-    }
-    case "invocation-result": {
-      assertResultMessage(message);
-      break;
-    }
-    default: {
-      assertGenericMessage(message);
-    }
-  }
-  return message;
-};
-
-export function hasInvocations(message: AgentMessage): boolean {
-  return (message.data.invocations && message.data.invocations.length > 0) ?? false;
-}
-
-export function assertAgentMessage(
-  message: Pick<RunMessage, "data" | "type">
-): asserts message is AgentMessage {
-  if (message.type !== "agent") {
-    throw new Error("Expected an AgentMessage");
-  }
-
-  const result = agentDataSchema.safeParse(message.data);
+const validateMessage = (message: unknown) => {
```

Comment:
Is this used?

Reply:
Yeah, in the insert new message and get messages.

Reply:
Removed that now.